schema/properties/easing-handle.json

```diff
@@ -36,17 +36,13 @@
                     "$ref": "#/$defs/values/vector",
```

Comment:
While looking at this, do you happen to know what's the use case for the array form of easing handle coordinates?

I seem to recall encountering single-value arrays in the wild, wondering if that's just legacy cruft or there's some legitimate use for multi-dimensional handle coordinates?  AFAIK easing curves are always two-dimensional, so their coefficients should be scalars.

Reply:
bodymoving exports them like that, I think the logic is:
* scalar -> handles are array with a single element
* position -> handles are a number
* multidimensional -> handles are arrays with _n_ elements